sdk/go/common/testing/environment.go

```diff
@@ -119,26 +120,29 @@ func (e *Environment) ImportDirectory(path string) {
 	}
 }
 
-// DeleteEnvironment deletes the environment's RootPath, and everything underneath it.
+// DeleteEnvironment deletes the environment's HomePath and RootPath, and everything underneath them.
 func (e *Environment) DeleteEnvironment() {
 	e.Helper()
-	err := os.RemoveAll(e.RootPath)
-	if err != nil {
-		// In CI, Windows sometimes lags behind in marking a resource
-		// as unused. This causes otherwise passing tests to fail.
-		// So ignore errors during cleanup.
-		e.Logf("error cleaning up test directory %q: %v", e.RootPath, err)
+	for _, path := range []string{e.HomePath, e.RootPath} {
+		if err := os.RemoveAll(path); err != nil {
+			// In CI, Windows sometimes lags behind in marking a resource
+			// as unused. This causes otherwise passing tests to fail.
+			// So ignore errors during cleanup.
+			e.Logf("error cleaning up test directory %q: %v", path, err)
+		}
 	}
 }
 
-// DeleteEnvironment deletes the environment's RootPath, and everything
-// underneath it. It tolerates failing to delete the environment.
+// DeleteEnvironment deletes the environment's HomePath and RootPath, and everything
```

Comment:
Why do we even have this? The normal Delete environment is tolerant anyway

Reply:
https://github.com/pulumi/pulumi/pull/19025